cpp/common/src/codingstandards/cpp/exceptions/ExceptionFlowCustomizations.qll

```diff
@@ -0,0 +1,113 @@
+/*
+ * A library customize models that model the flow of exceptions through the program.
+ */
+
+import cpp
+private import codingstandards.cpp.exceptions.ExceptionFlow
+
+/** A `ThrowingExpr` which is the origin of a exceptions in the program. */
+abstract class OriginThrowingExpr extends ThrowingExpr { }
+
+/**
+ * A `FunctionCall` to an external function without an exception specification that *
+ *  may throw an exception.
+ */
+abstract class ExternalUnderspecifiedFunctionCallThrowingExpr extends FunctionCall, ThrowingExpr { }
```

Comment:
I dont quite see the purpose of this addition (was this meant to be left in here, since you are actually using the extensible models approach instead?)

Reply:
This is here so it can be extended from QL as well if needed. Model as data is the preferred route, but has limitations due to its expressiveness.